client/profile-wizard/style.scss

```diff
@@ -91,10 +99,32 @@
 			font-weight: 400;
 		}
 
+		.components-popover__content {
+			white-space: pre;
+			font-size: 16px;
+
+			> div {
+				padding: 11px 16px;
+			}
+		}
+
 		.woocommerce-card ~ p {
 			font-size: 12px;
 			color: $core-grey-dark-150;
 		}
+
+		.woocommerce-profile-wizard__footer {
+			margin: 34px auto 0 auto;
```

Comment:
I think some bottom margin will be better. It looks very close to the edge of the screen.

![Screen Shot 2020-07-09 at 1 21 17 PM](https://user-images.githubusercontent.com/1922453/86986402-b5355980-c1e7-11ea-90a2-bec8d4cd42c0.png)

Reply:
@psealock Could you provide a fuller screenshot? The color of the link looks wrong, and it looks like your screenshot has the link inside the card which isn't right with the design, but maybe I'm just struggling with context because the screenshot is smaller.

Reply:
I see what you mean on a short screen, so I've added the same top and bottom margin to the footer to fix this. Not sure why the link looks wrong in your screenshot though 😄